irohad/synchronizer/synchronizer.hpp

```diff
@@ -39,7 +39,10 @@ namespace iroha {
 
       /**
        * Emit committed blocks
-       * Note: from one block received on consensus
+       * Note 1: from one block received on consensus
+       * Note 2: commit may be empty; in that case, it can contain empty
```

Comment:
Can empty commit contain non-empty pointer? If no then "can contain empty pointer" does not need "can" in it.